static/app/components/objectInspector.tsx

```diff
@@ -0,0 +1,47 @@
+import {ComponentProps, useMemo} from 'react';
+import {
+  chromeDark,
+  chromeLight,
+  ObjectInspector as ReactObjectInspector,
+} from 'react-inspector';
+import {useTheme} from '@emotion/react';
+
+import ConfigStore from 'sentry/stores/configStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+
+type Props = Omit<ComponentProps<typeof ReactObjectInspector>, 'theme'> & {
+  theme?: Record<string, any>;
+};
+
+function ObjectInspector({theme, ...props}: Props) {
+  const config = useLegacyStore(ConfigStore);
```

Comment:
I don't think component will re-render if you use ConfigStore directly (though this shouldn't happen too often)